lascli/parser/datasets.py

```diff
@@ -321,7 +321,7 @@ def create_documents(
     return dict(counter)
 
 
-def get_documents(las_client: Client, dataset_id, output_dir, num_threads, chunk_size):
+def get_documents(las_client: Client, dataset_id, output_dir, num_threads, chunk_size, max_elements):
```

Comment:
Is there a reason to use a different term here than what we're used to in the API?

```suggestion
def get_documents(las_client: Client, dataset_id, output_dir, num_threads, chunk_size, max_results):
```